cmd/edgex-ui-server/static/js/pages/appService.js

```diff
@@ -223,7 +223,7 @@ orgEdgexFoundry.appService = (function () {
             $.each(value,function (index,val) {
                 $("#appservice_plus"+key).append("<div class=\"helper-dialog-wrapper appservice_drop_card\" id = \""+key+"_"+val.Name+"\" title=\""+val.Name+"\">\n" +
                     "<div class=\"appservice_description\">\n" +
-                    "<h5 align=\"center\" class=\"appservice_transform\">"+val.Name+"</h5>\n" +
+                    "<h5 align=\"left\" class=\"appservice_transform\">"+val.Name+"</h5>\n" +
                     "</div>\n" +
                     "</div>");
```

Comment:
You should use single quotes on the outermost layer to combine your html code snippet,  better with variable declaration